packages/backend-defaults/src/entrypoints/cache/CacheManager.ts

```diff
@@ -140,37 +141,43 @@ export class CacheManager {
       );
     }
 
-    if (store === 'redis' || store === 'valkey') {
-      return CacheManager.parseRedisOptions(
-        store,
-        storeConfigPath,
-        config,
-        logger,
-      );
-    }
-
-    if (store === 'infinispan') {
-      return InfinispanOptionsMapper.parseInfinispanOptions(
-        storeConfigPath,
-        config,
-        logger,
-      );
+    switch (store) {
+      case 'redis':
+        return CacheManager.parseRedisOptions(
+          store,
+          storeConfigPath,
+          config,
+          logger,
+        );
+      case 'valkey':
+        return CacheManager.parseValkeyOptions(
+          store,
+          storeConfigPath,
+          config,
+          logger,
+        );
+      case 'infinispan':
+        return InfinispanOptionsMapper.parseInfinispanOptions(
+          storeConfigPath,
+          config,
+          logger,
+        );
+      default:
+        return undefined;
     }
-
-    return undefined;
   }
 
   /**
    * Parse Redis-specific options from configuration.
    */
   private static parseRedisOptions(
-    store: string,
+    store: 'redis',
```

Comment:
At this point let's just remove this argument. Same in the corresponding valkey method.